README.md

```diff
@@ -64,20 +64,20 @@ If you are using [OpenResty](http://openresty.org/), the default location would
 ## Support
 
 #### Community Support
-For generic questions, see the Wiki pages with Frequently Asked Questions at:  
```

Comment:
I deliberately want the link on a new line

Reply:
fixed